<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HysteresisGraph</class>
 <widget class="QWidget" name="HysteresisGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber_x">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_y">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_lcd_x">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>20</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lcd_y">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_axis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_axis">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>620</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>In</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_HysteresisGraph">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_max">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_y_min">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_min">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_x_max">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>620</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
